api/registry/registry_integration.go

```diff
@@ -35,14 +35,34 @@ func (d *idm) Init(ctx types.Context, config gofig.Config) error {
 	d.config = config
 	d.ctx = ctx
 	d.used = map[string]int{}
-	ctx.WithField("pathCache", "").Debug("checking volume path cache setting")
+	ctx.WithField(types.ConfigIntegrationVolPathCache, d.pathCache()).
+		WithField("serviceName", ctx.ServiceName()).Info(
+		"empty path for unused volume setting")
+	ctx.WithField(types.ConfigIntegrationVolUnmountIgnoreUsed,
+		d.ignoreUsedCount()).
+		WithField("serviceName", ctx.ServiceName()).Info(
+		"ignore used count on unmount setting")
+	ctx.WithField(types.ConfigIntegrationVolMountPreempt, d.preempt()).
+		WithField("serviceName", ctx.ServiceName()).Info(
+		"pre-emptive mount setting")
+	ctx.WithField(types.ConfigIntegrationVolCreateDisable, d.disableCreate()).
+		WithField("serviceName", ctx.ServiceName()).Info(
+		"volume creation disabled setting")
+	ctx.WithField(types.ConfigIntegrationVolRemoveDisable, d.disableRemove()).
+		WithField("serviceName", ctx.ServiceName()).Info(
+		"volume removal disabled setting")
+
```

Comment:
Hi @clintonskitson,

I've notices some variable scoping bugs in places. I'm finding them and fixing them. Using your example above:

```
 if config.IsSet(types.ConfigService) {
        ctx := ctx.WithServiceName(config.GetString(types.ConfigService))
}
```

I've seen stuff like that in places (they're my fault) and instead of updating the outer ctx with a new one, it's just creating a scoped ctx variable with the new value.